src/content/docs/ai-monitoring/compatibility-requirements-ai-monitoring.mdx

```diff
@@ -0,0 +1,71 @@
+
+## Compatible AI libraries [#compatibility] 
+
+Below you can check if you're using the correct agent version for AI monitoring. You can also confirm whether our APM agents can collect data from the libraries or frameworks you use in your AI-powered app.
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Agent version
+            </th>
+            <th>
+                Supported libraries
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+            [Go version 3.31.0 and above](/docs/apm/agents/go-agent/get-started/go-agent-compatibility-requirements/#digital-intelligence-platform)
+            </td>
+            <td>
+            * [Go OpenAI library](https://github.com/sashabaranov/go-openai) versions 3.4.0 and above
+            * [AWS SDK for Go v2](https://github.com/aws/aws-sdk-go-v2) versions 1.6.0 and above
+            </td>
+        </tr>
+        <tr>
+            <td>
+            [Node.js version 11.13.0 and above](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/#digital-intelligence-platform)
+            </td>
+            <td>
+            * [OpenAI Node.js API library](https://www.npmjs.com/package/openai/v/4.0.0-beta.4) versions 4.0.0 and above. If your model uses streaming, the Node.js agent supports versions 4.12.2 and above
+            * [AWS SDK for JavaScript BedrockRuntime Client](https://www.npmjs.com/package/@aws-sdk/client-bedrock-runtime) versions 3.474.0 and above
+            * [LangChain.js](https://www.npmjs.com/package/langchain/v/0.1.17) versions 0.1.17 and above
+            </td>
+        </tr>
+        <tr>
+            <td>
+            [Python version 9.9.0 and above](/docs/apm/agents/python-agent/getting-started/compatibility-requirements-python-agent/#digital-intelligence-platform)
+            </td>
+            <td>
+            * [OpenAI](https://pypi.org/project/openai/) library versions 1.13.3 and above
+            * [Boto3 AWS SDK for Python](https://docs.aws.amazon.com/pythonsdk/) 
+            * [LangChain](https://pypi.org/project/langchain/0.0.300/) versions 0.1.17 and above
+            </td>
+        </tr>
+        <tr>
+            <td>
+            [Ruby version 9.8.0 and above](/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks/#digital-intelligence-platform)
+            </td>
+            <td>
+            * [OpenAI gem](https://github.com/alexrudall/ruby-openai) version 3.4.0 and above
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+Certain SDKs may contain support for certain AI models. For example, Amazon BedRock supports Anthropic. If an SDK in the table contains a model but the table doesn't list the model itself, AI monitoring may still collect data about that model.
```

Comment:
I'm not sure I follow the second sentence here. Is there a separate place where we list the models that we do support? There have been some recent changes (in the last few weeks) where there are models that we do not support yet. This of course can be different per language as well.

Reply:
@bonczj From my understanding, BedRock supports certain models (like Claude for Anthropic among others) so we'd be able to "see" into the data that passes through that model vis a vis Bedrock (or LangChain) My understanding, though, about how this works specifically is hazier. We have a couple options:

1. We add a second h2 that explains for agents that support Bedrock and LangChain, they support [MODELS]. if we go this route, I would need a pretty detailed list of all available models and the versions that are supported via LangChain and Bedrock. I'm noot sure who would provide this list

2. We keep the doc as is, remove this call out you commented on, and add details postGA when teams have more capacity to get more granular.

What do you think?